.gitpod.yml

```diff
@@ -1,26 +1,38 @@
 # List the start up tasks. Learn more https://www.gitpod.io/docs/config-start-tasks/
 tasks:
-  - name: Uninstall lynx
-    init: sudo apt-get remove lynx -y
-  - name: Install AWS CLI
+  - name: Gitpod config (browser open, workspace bin path)
     init: |
-      curl "https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip" -o "awscliv2.zip"
-      unzip awscliv2.zip
-      sudo ./aws/install
-      rm awscliv2.zip
-      rm -rf ./aws
-    command: aws --version
-
-  - name: Install Conveyor CLI
+      mkdir -p /workspace/bin
+      cat > /workspace/bin/open.sh <<'EOF'
+      #!/bin/bash
+      exec gp preview --external "$@"
```

Comment:
What does this do? Enable a preview of some sorts?

Reply:
`gp` is the gitpod helper pre-installed in gitpod
`gp preview --external` will make gitpod open a browser window